Sources/SwiftLayout/Extension/Kit/ViewAlias.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  View+Kit.swift
+//  
+//
+//  Created by oozoofrog on 2022/03/11.
+//
+
+import Foundation
+
+#if canImport(AppKit)
+import AppKit
+public typealias UIView = AppKit.NSView
```

Comment:
It doesn't seem like a good idea to typealias `NSView` directly to `UIView`.\
In my opinion, it is better to define a new type like `SLView` (SwiftLayout View) or `NSUIView`.

If you do this, it is expected that the following imports can also be removed from some files.
```swift
#if canImport(UIKit)
import UIKit
#elseif canImport(AppKit)
import AppKit
#endif
 ```

Reply:
@gmlwhdtjd thank you. you right absolutely. and  have something to confess, there was be SwiftLayoutView on first commit of pr. it was removed by my mistook, and leave from us. so, i swear here on m1 max Mac book (of future), to get that code back from Code Underworld.

Reply:
@gmlwhdtjd show our SL*!